en/LLSEPluginDevelopment/ScriptAPI/LLSEJSPlugin.md

```diff
@@ -0,0 +1,60 @@
+# LLSE - Using Javascript to Create Your First Plugin
+
+> This guide serves to demonstrate the very simple and straight forward process to creating your first plugin - as well as some best practices when thinking about what to make and how to do it. Experience using Javascript previously is recommended, but not required. Javascript is a very beginner friendly language, so don't get overwhelmed!
+
+## Prerequisites
+
+Before developing your first plugin, we will need to set up your developer environment (IDE). The choice of software you use to program is yours, but trusted and reliable software is recommended. [Atom](https://atom.io/) is a very simple program which can be used for this purpose.
+
+You will also need to set up a clean LiteLoaderBDS installation, details on how to install LiteLoaderBDS can be found [here](https://github.com/LiteLDev/LiteLoaderBDS#-install). This server will be used to test your plugin.
```

Comment:
Here you should write the link to the documentation (https://docs.litebds.com/en/#/Usage).